ts/testUtils.ts

```diff
@@ -60,7 +60,9 @@ export const runExecutable = (
   executable: garn.Executable,
   options: { cwd?: string } = {},
 ): Output => {
-  const tempDir = Deno.makeTempDirSync({ prefix: "garn-test" });
+  const tempDir = options.cwd
+    ? options.cwd
+    : Deno.makeTempDirSync({ prefix: "garn-test" });
```

Comment:
This is causing the failing tests since the gh-pages tests are assuming they run in an empty directory. We can make this work, but in order to do so we'll need to
* `git add flake.nix` in this function
* change expectations to include `flake.nix`, and `flake.lock` in the gh-pages test assertions
* append `-dirty` to the end of the git shas in the output.

Maybe it makes sense to have a separate option for the gh-pages tests to keep the current behavior?

Reply:
Nit, if we do keep this as it is
```suggestion
  const tempDir = options.cwd ?? Deno.makeTempDirSync({ prefix: "garn-test" });
```

Reply:
Yeah, not sure what to do. My first thought was to allow absolute paths in project sources, since then we could have the project point to wherever the source is. But I think that breaks a lot of assumptions and shouldn't be allowed in user code, at least. Ultimately I think the flake file *has* to be in the same directory as the source.

I think you're idea of a separate option makes sense.

Reply:
Fixed. Let me know what you think.